### DISCUSSION TITLE
Added some gpt plugins and plugin management crashes

### DISCUSSION THREAD

**Discussion by RO-mix** [LAION-AI/Open-Assistant#3362] [Q&A]
Added some gpt plugins and plugin management crashes. I think because of some parts of installed plugs...
------ browser log -------
Uncaught Error: Permission denied to access property "matches"
    S index.esm.js:278
    M index.esm.js:333
    k index.esm.js:310
    L index.esm.js:597
  
  How to reset plugin list??

**Comment by andreaskoepf** [comment]
Could you please provide the URL of the plugin you tried to add? This would allow us to analyze and potentially fix the problem. The selected plugin-configuration is stored in the local storage of your browser see: [chat.ts#L17C1-L31 ](https://github.com/LAION-AI/Open-Assistant/blob/70f30a6d6f23551a5139711f84fd58eefc4fe288/website/src/utils/chat.ts#L17C1-L31) By deleting the local storage of open-assistant.io in your browser you can hard 'reset' the chat configuration.
